gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Nit: looks like an IDE injected this, and needs to be updated or removed.

Reply:
this is gradle's script, not ours

Reply:
(specifically this is autogenerated when you run `gradlew wrapper` to update to the latest gradle version, so I don't really want to deviate from what that produces)